packages/hardhat-vyper/src/compilation.ts

```diff
@@ -52,11 +52,11 @@ export async function compile(
     const pathFromSources = path.relative(paths.sources, file);
 
     if (await isAlreadyCompiled(file, paths, vyperVersion, files)) {
-      console.log(pathFromCWD, "is already compiled");
+      console.log("✓", pathFromCWD, "is already compiled");
       continue;
     }
 
-    console.log("Compiling", pathFromCWD);
+    console.log("⌛ Compiling", pathFromCWD);
```

Comment:
I would avoid doing this, since some terminals don't handle emojis well.

Reply:
It's hard to distinguish between what is compiling and what already has compiled without the emoji - It's a huge wall of white text. I made the change based on the way mocha displays green ticks next to passed tests.

Reply:
I understand, but the problem is that it looks like this in a terminal that doesn't support emojis:
![image](https://user-images.githubusercontent.com/417134/102532191-5cb19480-4082-11eb-9968-245d99928fcd.png)

Maybe making the "Nothing to compile" output greent would help? (`chalk` handles correctly terminals without color support, and in any case that's less common that not having emoji support)

@alcuadrado what do you think?

Reply:
We have a function, `emoji`, that can be used for this. Here's an example: https://github.com/nomiclabs/hardhat/blob/aaa9cfe6a590ef578432c0d66e7dc58517ef65bb/packages/hardhat-core/src/internal/cli/project-creation.ts#L84